parser.y

```diff
@@ -7643,6 +7662,22 @@ TableOption:
 		yylex.AppendError(yylex.Errorf("The STORAGE clause is parsed but ignored by all storage engines."))
 		parser.lastErrorAsWarn()
 	}
+|	"SECONDARY_ENGINE" EqOpt "NULL"
+	{
+		// Parse it but will ignore it
+		// See https://github.com/mysql/mysql-server/blob/8.0/sql/sql_yacc.yy#L5977-L5984
+		$$ = &ast.TableOption{Tp: ast.TableOptionSecondaryEngine, StrValue: ""}
```

Comment:
`SECONDARY_ENGINE = NULL` and `SECONDARY_ENGINE = ''` have two different meanings and shouldn't be mixed.

Reply:
It should be in another pr, I'll revert it